package.json

```diff
@@ -6,5 +6,23 @@
     "type": "git",
     "url": "https://github.com/EdHintz/confirmatic.git"
   },
-  "scripts": {"test": "grunt"}
-}
\ No newline at end of file
+  "scripts": {
+    "test": "grunt",
+    "start": "node lib/server.js"
+  },
+  "devDependencies": {
+    "chai": "^2.2.0",
+    "grunt": "^0.4.5",
+    "grunt-cli": "^0.1.13",
+    "grunt-contrib-clean": "^0.6.0",
+    "grunt-contrib-jshint": "^0.11.1",
+    "grunt-contrib-watch": "^0.6.1",
+    "grunt-jscs": "^1.6.0",
+    "grunt-mocha-istanbul": "^2.3.1",
+    "sinon": "^1.14.1"
+  },
+  "dependencies": {
+    "node-bandwidth": "^1.0.7",
+    "q": "^1.2.0"
```

Comment:
It doesn't look like `q` is currently used in this project (I may have missed it). In any case, we have started using (Bluebird)[https://github.com/petkaantonov/bluebird] on newer incubation projects. Might be worth considering (the API is a little more similar to the native Promise API in newer runtimes).